examples/cloudrun/wait-approval/.pipe.yaml

```diff
@@ -0,0 +1,10 @@
+# Deployment pipeline that contains a manual approval stage.
+apiVersion: pipecd.dev/v1beta1
+kind: CloudRunApp
+spec:
```

Comment:
Because all examples could be used to deploy to somewhere so we should have a pipeline with a WAIT_APPROVAL stage. For example, we can use this pipeline by replacing `WAIT` with `WAIT_APPOVAL`
https://github.com/pipe-cd/pipe/blob/master/examples/cloudrun/canary/.pipe.yaml#L5-L24

Reply:
5cb3bfe